tests/conftest.py

```diff
@@ -39,8 +39,8 @@
     "[A ; B]ff",
     "[A*]tt",
     "[A*]ff",
-    "[<A>tt?]tt",
-    "[<A>tt?]ff",
+    "[?<A>tt]tt",
+    "[?<A>tt]ff",
```

Comment:
Every formula in this file only use `tt` and `ff` as final check. Maybe we should add (or change some with) new such as `<a>b`.

Reply:
yes. Although formulas like `<a>b` should be parsed as `<a><b>tt`

Reply:
Right, I've added few tests in test_truth(), leaving this file as is.